scripts/travis/build-all-in-one-image.sh

```diff
@@ -34,18 +42,54 @@ upload_to_docker() {
     echo 'skip docker upload for PR'
     exit 0
   fi
-  export REPO=$1
-  bash ./scripts/travis/upload-to-docker.sh
+
+  REPO=${1}
+  CMD_ROOT=${1}
```

Comment:
both of these are `$1`?

Reply:
Sorry. CMD_ROOT should be `$2`. fixed it.